doc/user_guide/marks/geoshape.rst

```diff
@@ -630,3 +639,9 @@ We use here an elegant way to access the nested point coordinates from the geome
 
 The earthquakes are displayed using a ``mark_geoshape`` and filtered once out of sight of
 the visible part of the world. A hover highlighting is added to get more insight of each earthquake.
+
+Tile-based Maps
+^^^^^^^^^^^^^^^
+
+To use tile-based maps (such as OpenStreetMap) as the background for ``mark_geoshape``,
+you can use the package `Altair Tiles <https://altair-viz.github.io/altair_tiles>`_ together with Altair.
```

Comment:
@binste Now that `altair_tiles` is part of `altair[all]` I would be onboard with adding some of the examples directly into this mark_geoshape page if you think that makes sense (as well as a few gallery examples for visibility).

Reply:
Thanks for linking to altair_tiles! Yeah, sounds like a good next step to add a simple example to the Altair docs. I have it on my list :)